lib/diagnostic.js

```diff
@@ -17,7 +17,13 @@
 // Methods: sell, call
 
 // Answer Starts Here
-//
+
+let forCar = function(make, model, year){
+  this.make = make;
+  this.model = model;
+  this.year = year;
+
+}
```

Comment:
We want two examples of entities along with their attributes and methods for a browser-based calendar application.